buildenv

```diff
@@ -0,0 +1,37 @@
+export ZOPEN_TARBALL_URL="https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.2.1.tar.bz2"
+export ZOPEN_TARBALL_DEPS="make bzip2 tar curl perl automake autoconf m4 sed gettext zoslib diffutils libassuan libgpgerror"
+export ZOPEN_TYPE="TARBALL"
+export ZOPEN_COMP=CLANG
+export ZOPEN_CONFIGURE_OPTS=" --prefix=/home/haritha/zopen/prod/pinentry-1.2.1/ --enable-pinentry-tty"
+
+PINENTRY_VERSION="1.2.1"
```

Comment:
Suggest moving this up, and using it in the tarball url so that you only have to change one place.

Reply:
Right, i have done it, Igor! Thank you!

> approve pending igor's suggestions

Sure, Thanks Mike!